src/cli/cli_coap.cpp

```diff
@@ -119,6 +175,55 @@ otError Coap::ProcessResource(int argc, char *argv[])
     return OT_ERROR_NONE;
 }
 
+otError Coap::ProcessSet(int argc, char *argv[])
+{
+    otMessage *   notificationMessage = NULL;
+    otMessageInfo messageInfo;
+    otError       error = OT_ERROR_NONE;
+
+    if (argc > 1)
+    {
+        VerifyOrExit(strlen(argv[1]) < (kMaxBufferSize - 1), error = OT_ERROR_INVALID_ARGS);
+        strlcpy(mResourceContent, argv[1], kMaxBufferSize);
```

Comment:
Prefer using `strncpy` - we are trying to move away from `strlpy` (see https://github.com/openthread/openthread/commit/708c250fe5ef7057e6395d9759fb341d150e89e8).

Reply:
Yep, I'll fix that… there's two other places but those are probably beyond the scope of this PR.
* `src/cli/cli_coap_secure.cpp`
* `src/core/meshcop/commissioner.cpp`